frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -73,7 +73,7 @@ export function OperatorValueSelect({
 }: OperatorValueSelectProps): JSX.Element {
     const propertyDefinition = propertyDefinitions.find((pd) => pd.name === propertyKey)
 
-    const isCohortProperty = propertyKey === 'id'
+    const isCohortProperty = propertyKey === 'id' && type === PropertyFilterType.Cohort
```

Comment:
this condition feels a bit weak because other properties can have the key be `id` (e.g., group properties have the `id` key, but they're not cohort filters).  Feels safer to add the filter type too, bc at this point we should have that data.

Can't tell if this is risky, but cc @aspicer since you've been in the property filter UI zone recently.